src/main/java/org/broadinstitute/hellbender/tools/AnalyzeSaturationMutagenesis.java

```diff
@@ -131,6 +131,9 @@ public final class AnalyzeSaturationMutagenesis extends GATKTool {
     @Argument(doc = "paired mode evaluation of variants (combine mates, when possible)", fullName = "paired-mode")
     private static boolean pairedMode = true;
 
+    @Argument(doc = "don't discard disjoint mates (combine variants from both reads)", fullName = "dont-ignore")
+    private static boolean noIgnore = false;
```

Comment:
This is fussy silliness, but could you change this to a somewhat more description name.  Something like noIgnoreDisjointPairs.

Reply:
Agreed and added.